apps/api/src/utils/resume_handler.py

```diff
@@ -11,20 +11,29 @@
 
 log = getLogger(__name__)
 
-RESUMES_FOLDER_ID = os.getenv("RESUMES_FOLDER_ID")
+HACKER_RESUMES_FOLDER_ID = os.getenv("HACKER_RESUMES_FOLDER_ID")
+MENTOR_RESUMES_FOLDER_ID = os.getenv("MENTOR_RESUMES_FOLDER_ID")
 SIZE_LIMIT = 500_000
 ACCEPTED_TYPES = ("application/pdf",)
 
 
 class Person(Protocol):
     first_name: str
     last_name: str
+    application_type: Union[Literal["Hacker"], Literal["Mentor"]]
 
 
 async def upload_resume(person: Person, resume_upload: UploadFile) -> HttpUrl:
     """Upload resume file to Google Drive and provide url to uploaded file.
     Reject files larger than size limit"""
-    if not RESUMES_FOLDER_ID:
+
+    RESUME_FOLDER_ID = None
+    if person.application_type == "Hacker":
+        RESUME_FOLDER_ID = HACKER_RESUMES_FOLDER_ID
+    elif person.application_type == "Mentor":
+        RESUME_FOLDER_ID = MENTOR_RESUMES_FOLDER_ID
```

Comment:
Suggestion: to ensure we don't miss a literal value if the input type is updated in the future, it would help to do [exhaustiveness checking](https://mypy.readthedocs.io/en/stable/literal_types.html#exhaustiveness-checking) with `assert_never`, e.g.
```python
else:
    assert_never(person.application_type)
```